src/garage/torch/algos/vpg.py

```diff
@@ -107,34 +123,28 @@ def _check_entropy_configuration(self, entropy_method, center_adv,
 
     def train_once(self, itr, paths):
         """Perform one step of policy optimization."""
-        samples_data = self.process_samples(itr, paths)
+        valids, obs, actions, rewards, average_return = self.process_samples(
+            itr, paths)  # noqa: E501
 
-        self._optimize_policy(itr, paths)
-        self.baseline.fit(paths)
-
-        return samples_data['average_return']
+        loss = self._compute_loss(itr, paths, valids, obs, actions, rewards)
```

Comment:
this is a nit, but i think `mean()` should be here. mean is part of the loss function.

Reply:
fixed